### ISSUE TITLE
Indexer: zooqle

### ISSUE THREAD

**Issue by Sidlegionair** [Sonarr/Sonarr#1392]
Hi guys,

I was wondering if there is a possibility you could add Zooqle to the indexers, they have a documented API so it should be do-able.

https://zooqle.com/help/api.html

Thanks in advance

**Comment by markus101** [comment]
Do they support searching? Based on what I can see they have RSS feeds (which we may be able to apply filters to) and their API is only there to look up information if you already have the torrent hash. We're not likely to implement it if its purely string based searching (no way to search by TVDB ID) as poorly named releases are incredibly painful to deal with and lead to a lot of support questions ("Sonarr can't find it, but I see it on the site").

**Comment by uzegonemad** [comment]
I took some time to investigate Zooqle and I'm not sure that it's feasible to integrate. However, take this information with a grain of salt.

**tl;dr;** there is RSS searching, but it's string-based and there's no way to search by TVDB ID.

-----

It is possible to browse an RSS feed although it is string-based. Any search term can be output as RSS if you add `fmt=rss` to the query string.

The search does support native filtering min/max file size, however I was unable to find a way to search by TVDB ID. (Synopses don't appear to be coming from TVDB or IMDB, either.)

Example search:
https://zooqle.com/search?q=mr+robot+500MB-1000MB+category%3ATV

as XML:
https://zooqle.com/search?q=mr+robot+500MB-1000MB+category%3ATV&fmt=rss

Content-type is `text/xml`.

If you create an account, you can "subscribe" to TV shows and the site will generate an RSS feed based on your subscriptions and feed configuration.

<img width="694" alt="screen shot 2016-07-25 at 10 51 18 pm" src="https://cloud.githubusercontent.com/assets/430255/17126176/49e60f2a-52c0-11e6-9ed3-2c0a63e9ffe7.png">

The generated RSS feed the same schema but the content-type is `applicaton/rss+xml`.

Link to my generated RSS feed: https://zooqle.com/rss/tv/j0v4e68ng.rss

Finally, unrelated to Sonarr but potentially a deterrent for some users: they appear to be blocking all of Private Internet Access's US and CA endpoints. I had to switch my VPN to Mexico to browse.

**Comment by kingy444** [comment]
Has anyone got this to work yet? I would be more than happy to just work with an RSS feed, but unfortunately even something like the above rss https://zooqle.com/rss/tv/j0v4e68ng.rss throws an indexer not supported error despite it being standard rss?
![image](https://user-images.githubusercontent.com/7148670/31814293-e82e5142-b5d4-11e7-8558-a971a82e3e05.png)

**Comment by Gwend4l** [comment]
I have not tried it myself, but you could set up a Jackett instance next to your Sonarr instance, and use Jackett to fetch all the torrent information. It seems that Jackett is compatible with Zooqle, according to the README: https://github.com/Jackett/Jackett

**Comment by markus101** [comment]
No plans to support Zooqle for searching, but RSS appears to work alright at this time, otherwise Jackett is available.

**Comment by 78Alpha** [comment]
Zooqle support was recently dropped, since zooqle changed how it works. Jackett can't work with zooqle using its own tools/methods
